api/src/main/java/edu/wpi/first/shuffleboard/api/util/NetworkTableUtils.java

```diff
@@ -132,7 +132,7 @@ public static DataType dataTypeForEntry(String key) {
       return DataTypes.Map;
     }
     if (rootTable.containsKey(normalKey)) {
-      return DataType.forJavaType(rootTable.getValue(normalKey).getClass());
+      return DataType.forJavaType(rootTable.getValue(normalKey, null).getClass());
```

Comment:
It would have thrown an exception any time this returns null. Because of the containsKey, this should be fine (other then not thread safe and potentially now has a race condition)